pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.flake8]
+# TODO add python directory here in later PR
```

Comment:
Any reason why `./python` is not included now? Maybe `./exampleCourse` and `./graders` as well. It's such a disruptive PR, may as well do it at once. Also note that `./question-servers` is not necessary, it used to be for when `./python` was called `./question-servers/freeformPythonLib`.

Reply:
`./python` was not included before (you can check the old deleted config). I'm fine running black / isort over everything at once, but there's changes outside of running those required for `./python` to pass a lint check (there are issues like unused imports, improper comparisons, etc). So we can run them over those files, but can't set up enforcement yet.

I didn't know about the question servers thing though, I'll go ahead and remove.

Reply:
@jonatanschroeder changes to docs are here: https://github.com/PrairieLearn/PrairieLearn/pull/6794

Reply:
@eliotwrobson I'd be fine to leave formatting + linting `./python` to another PR, but can we do that as a very quick followup? I'd like for all Python to be treated the same.

Reply:
Sounds like a plan! There are a couple litter carveouts that are needed, but not too bad otherwise. I just wanted to avoid making this PR more disruptive than necessary.

I'm traveling right now, but once this is merged I should be able to bang out that next one in a couple hours.